README.md

```diff
@@ -1,2 +1,122 @@
-# melita-io
-Module for communication with melita.io API for devices
+# Melita IO
+
+|                |                            |
+| -------------- | -------------------------- |
+| Name           | Melita IO                  |
+| Version        | v1.0.0                     |
+| Dockerhub Link | [weevenetwork/melita-io]() |
+| Authors        | Mesud Pasic                |
+
+- [MQTT Ingress](#melita-io)
+  - [Description](#description)
+  - [Features](#features)
+  - [Environment Variables](#environment-variables)
+    - [Module Specific](#module-specific)
+    - [Set by the weeve Agent on the edge-node](#set-by-the-weeve-agent-on-the-edge-node)
+  - [Dependencies](#dependencies)
+
+## Description
+
+Melita.IO API module.
+
+## Features
+
+- Parsing Melita.io data for thermostat
+- Sends data to next module service via REST API
+
+## Environment Variables
+
+- INGRESS_HOST
+- INGRESS_PORT
+- EXECUTE_SINGLE_COMMAND
+- SINGLE_COMMAND
```

Comment:
instead of this add the table:

| Environment Variables | type | Description |
| --- | --- | --- |
| EXECUTE_SINGLE_COMMAND | string | yes/no to specify if module supports only executing single command |
| SINGLE_COMMAND | string | if EXECUTE_SINGLE_COMMAND=no, then users will have to provide in request "command" parameter with command name |
| MELITA_API_URL | string | Melita.io API URL |
| AUTHENTICATION_API_KEY | string | API key for authentication |
| ERROR_URL | string | URL for passing errors during request to melita.io API |